libkbfs/conflict_resolver.go

```diff
@@ -220,13 +226,29 @@ func (cr *ConflictResolver) getMDs(ctx context.Context, lState *lockState) (
 	if err != nil {
```

Comment:
Ok, I've added function comments in `md_util.go` and `folder_branch_ops.go` to clarify things, with a TODO.